### ISSUE TITLE
Segmentation fault for signing application bundle

### ISSUE THREAD

**Issue by vk8a8** [Dadoum/Sideloader#46]
`Signing the application bundle |########################        | 77/100            cli_frontend ERROR 2024-04-29T10:45:15.607 object.Exception at std/file.d:4101: Failed to stat file '/tmp/Spotify-8.5.60_Spotilife-1.8-Revised v2.ipa/Payload/Spotify.app/PlugIns/NotificationContentExtension.appex/SC_Info/NotificationContentExtension.sinf' Segmentation fault (core dumped) `

this is CLI on x86_64 arch linux

sorry for my persistent issues.

**Comment by Dadoum** [comment]
No problem about the issues. Software has to be improved, especially if it has problems.

I still can't code right now, but as a workaround open your ipa as a zip and remove every folder called SC_Info. That's data remaining from iPhone dumps to identify the account/the phone who originally installed it and prevent piracy. Sideloaders should remove it automatically, but here for some reason it fails. However if you remove it yourself from the file  it should work fine (let the issue open I will fix it when I'll get more time).

**Comment by vk8a8** [comment]
should/can i rezip it to ipa? and how?

**Comment by Dadoum** [comment]
Yes you can. When you unzip an ipa file you should see a Payload folder. Zipping it back in a new archive just makes a new iOS. It's that simple

**Comment by vk8a8** [comment]
do i need some sort of package to be able to zip it to ipa? or is it ok to just put it in a zip

**Comment by Dadoum** [comment]
A simple zip will do the trick, but you will probably have to rename that zip and change the end to ipa

**Comment by vk8a8** [comment]
hm, i get the same error again after removing the folders.

**Comment by vk8a8** [comment]
i've retried it a coupel times, and it seems to randomly flip between
`Signing the application bundle |########################        | 77/100Segmentation fault (core dumped)`,

`cli_frontend ERROR 2024-04-29T19:12:11.027 std.file.FileException at /opt/hostedtoolcache/dc/ldc2-1.33.0/x64/ldc2-1.33.0-linux-x86_64/bin/../import/std/file.d:2061: /tmp/Payload.ipa/Payload/Spotify.app/PlugIns/MessagesExtension.appex/_CodeSignature/CodeResources: No such file or directory`,
and being stuck on 80%.

**Comment by vk8a8** [comment]
it randomly start working? don't know what i did.
